### PR TITLE
[MINOR] docs: Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by coolderli**
### What changes were proposed in this pull request?

- Add Peidian Li to the contributor list.

### Why are the changes needed?

Fix: #3361 

### Does this PR introduce _any_ user-facing change?

- no

### How was this patch tested?

- no need.

### CODE REVIEW

**Comment by coolderli** [comment]
@justinmclean

File: common/src/main/resources/project.properties
Comment (zhoukangcn) [inline common/src/main/resources/project.properties:1]: Should be removed ?

Comment (justinmclean) [reply common/src/main/resources/project.properties:1]: there's no need for the extra blank line - no issue if it is removed

**Review by justinmclean — DISMISSED** [review]
Thanks for adding your name

File: common/src/main/resources/project.properties
Comment (coolderli) [reply common/src/main/resources/project.properties:1]: Removed

**Review by jerryshao — APPROVED** [review]

**Pull request merged**